src/spec/settings_spec.js

```diff
@@ -16,25 +16,31 @@ describe('Settings Page', () => {
         auth.saveToken(permanentToken);
       }));
   }
-  ///This test will be commented until we add the new Domain Functionality
-  //it('should show add domain input when you click in add domain button', () => {
-    // Arrange
-    //beginAuthenticatedSession();
-    //var settings = new SettingsPage();
+  it('should show add domain input when you click in add domain button', () => {

```

Comment:
I do not understand what the word `input` mean in this context.

Maybe it is referring to the text added, but it makes me think in a input element.

What do you think about to simply remove that word?